src/controls/pagination/Pagination.tsx

```diff
@@ -3,6 +3,7 @@ import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button'
 import styles from "./Pagination.module.scss";
 import { isEqual } from "lodash";
 import * as telemetry from '../../common/telemetry';
+import { Stack } from "office-ui-fabric-react";
```

Comment:
Why do we have `Pagination` included in this PR? Is this intended?
If yes - please, update OUIFR import.

Reply:
The pagination control with the update of the version the buttons were misaligned, I made a small adjustment.